### PR TITLE
Add Release notes for KubeDB v2024.9.30

### PR DESCRIPTION
**Pull request by AshrafulHaqueToni**

### CODE REVIEW

File: content/post/kubedb-v2024.9.30/index.md [hunk 1]
```diff
@@ -0,0 +1,979 @@
...
+            storage: 1Gi
+  deletionPolicy: WipeOut
+``` 
+
+## Druid
+This release introduces support for comprehensive disaster recovery of `Druid` clusters with Stash 2.0, also known as Kubestash. Kubestash, offered by AppsCode, provides a cloud-native backup and recovery solution for Kubernetes workloads, streamlining operations through its operator-driven approach.  It works with any existing KubeDB or Stash license key.
+
+You need to create the following resources to successfully take a backup and restore a `Druid` cluster.
+
+BackupStorage: refers to a cloud storage backend (like `s3`, `gcs` etc.) you prefer.
```
Comment (tapojit047) [inline content/post/kubedb-v2024.9.30/index.md:161]: Make BackupStorage, RetentionPolicy, Secrets, BackupConfiguration, RestoreSession **bold**

File: content/post/kubedb-v2024.9.30/index.md [hunk 2]
```diff
@@ -0,0 +1,979 @@
...
+
+RetentionPolicy: allows you to set how long you’d like to retain the backup data.
+
+Secrets: hold the backend access credentials and a Restic encryption password to encrypt the backup snapshots.
+
+BackupConfiguration: specifies the target database, backend, addon etc. to use to back up the database.
+
+RestoreSession: specifies the target database, addon, datasource to use to restore the backed up data. Here is an example of BackupConfiguration
```
Comment (tapojit047) [inline content/post/kubedb-v2024.9.30/index.md:169]: Remove "Here is an example of BackupConfiguration"

**Review by tapojit047 — CHANGES_REQUESTED** [review]

**Review by sheikh-arman — CHANGES_REQUESTED** [review]
Change the `Internally Managed ClickHouse Keeper` heading
to `Managed ClickHouse Keeper`

File: content/post/kubedb-v2024.9.30/index.md [hunk 3]
```diff
@@ -0,0 +1,979 @@
...
+
+```yaml
+apiVersion: core.kubestash.com/v1alpha1
+kind: RestoreSession
+metadata:
+  name: restore-sample-singlestore
+  namespace: demo
+spec:
+  manifestOptions:
+    restoreNamespace: dev
```
Comment (ishtiaqhimel) [inline content/post/kubedb-v2024.9.30/index.md:756]: `restoreNamespace` should be the sub-field of `singlestore`.
```
singlestore:
  restoreNamespace: dev
```

Also do not need to specify `db: true`. Its default value is true.

**Review by ishtiaqhimel — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2024.9.30/index.md [hunk 4]
```diff
@@ -0,0 +1,1036 @@
...
+- **Autoscaling**: Added autoscaling support for FerretDB and Microsoft SQL Server to automatically adjust resources based on workload demands.
+  
+- **Network Policies**: We have added support for [NetworkPolicy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) in the release.Now Users can pass
+`--set global.networkPolicy.enabled=true` while installing KubeDB. The required Network policies for operators will be created as part of the release process. And the required Network policies for DB will be created by the operator when some database gets provisioned.
+
+- **Backup & Restore**: Comprehensive disaster recovery support for Druid clusters using Kubestash (Stash 2.0), and manifest backup support for SingleStore.
+
+- **Managed ClickHouse Keeper**: Introduced support for Internally Managed ClickHouse Keeper and KubeDB Managed ClickHouse Keeper Server, expanding options for ClickHouse management.
+
+- **New Version Support**: Added support for MySQL version 8.4.2 and Microsoft SQL Server 2022-cu14.
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.9.30/index.md:59]: highlight versions

File: content/post/kubedb-v2024.9.30/index.md [hunk 5]
```diff
@@ -0,0 +1,1036 @@
...
+        replicas: 3
+        storage:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+  deletionPolicy: WipeOut
+ ```
+Supported Versions: 4.1.6 & 5.0.0
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.9.30/index.md:88]: highlight versions

File: content/post/kubedb-v2024.9.30/index.md [hunk 6]
```diff
@@ -0,0 +1,1036 @@
...
+
+
+## ClickHouse
+
+In this release added `Internally Managed ClickHouse Keeper`, `KubeDB Managed ClickHouse Keeper Server` support. Previously only `Externally Managed ClickHouse Keeper` was supported. This release also added Custom Configuration and Monitoring Support.
+
+
+You need to set spec.clusterTopology.clickHouseKeeper.externallyManaged field as false to use KubeDB managed clickhouse keeper.
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.9.30/index.md:96]: highlight `spec.clusterTopology.clickHouseKeeper.externallyManaged`

File: content/post/kubedb-v2024.9.30/index.md [hunk 7]
```diff
@@ -0,0 +1,1036 @@
...
+`RestoreSession`: specifies the target database, addon, datasource to use to restore the backed up data. Here is an example of BackupConfiguration
+
+You can take both Logical and Manifest backup of druid clusters separately. Or both together which we call Application Level backup
+
+where we take Application Level backup:
+
+
+```
+apiVersion: core.kubestash.com/v1alpha1
+kind: BackupConfiguration
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.9.30/index.md:178]: use ```yaml. also update all if missing

File: content/post/kubedb-v2024.9.30/index.md [hunk 8]
```diff
@@ -0,0 +1,1036 @@
...
+
+We are thrilled to announce the release of **KubeDB v2024.9.30**. This release introduces several key features, including:
+
+- **Added New Database Support**: Added new database support Apache Cassandra, broadening our database management capabilities.
+
+- **OpsRequest Support**: Enhanced operational request capabilities for FerretDB, Microsoft SQL Server, PgBouncer, Solr, and ZooKeeper, providing greater management flexibility.
+  
+- **Autoscaling**: Added autoscaling support for FerretDB and Microsoft SQL Server to automatically adjust resources based on workload demands.
+  
+- **Network Policies**: We have added support for [NetworkPolicy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) in the release.Now Users can pass
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.9.30/index.md:52]: need a space before `Now Users`

**Review by obaydullahmhs — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2024.9.30/index.md [hunk 9]
```diff
@@ -0,0 +1,1034 @@
+---
+title: Announcing KubeDB v2024.9.30
+date: "2024-09-30"
+weight: 14
+authors:
+- Ashraful Haque Tani
```
Comment (mohin7) [inline content/post/kubedb-v2024.9.30/index.md:6]: Update with `Ashraful Haque`

**Pull request merged**